src/Evolution/Systems/GrMhd/ValenciaDivClean/Sources.cpp

```diff
@@ -22,53 +22,72 @@
 
 /// \cond
 namespace {
-tnsr::II<DataVector, 3, Frame::Inertial> densitized_stress(
+void densitized_stress(
+    const gsl::not_null<tnsr::II<DataVector, 3, Frame::Inertial>*> result,
+    const gsl::not_null<tnsr::I<DataVector, 3, Frame::Inertial>*> temp_buffer3,
```

Comment:
Meant to indicate the number of allocations available since I used it for 3 scalars. I'm not at all attached to the name so if you prefer anything else that's more meaningful I'm happy to change it. Maybe `temp_buffer_3_scalars`?